server/game/Effects/StaticEffect.js

```diff
@@ -84,6 +98,18 @@ class StaticEffect {
         return !hasDash[this.type] || !hasDash[this.type](target, this.value);
     }
 
+    isMilitaryModifier() {
+        return MilitaryModifiers.includes(this.type);
+    }
+
+    isPoliticalModifier() {
+        return PoliticalModifiers.includes(this.type);
+    }
+
+    isModifier() {
```

Comment:
Maybe `isSkillModifier` might be a more appropriate name?  (To distinguish from glory modfiers)